packages/web/src/components/create-playlist/PlaylistForm.module.css

```diff
@@ -0,0 +1,70 @@
+.playlistForm {
+  display: flex;
+  align-items: stretch;
+}
+
+.playlistForm > .form {
```

Comment:
nit: I'm not sure this needs to be this specific. `.form` is only used in one place?